etc/nebula-metad.conf.default

```diff
@@ -34,6 +34,6 @@
 
 ########## Misc #########
 # The default number of parts when a space is created
---default_parts_num=1024
+--default_parts_num=100
```

Comment:
What's wrong with too many partitions?

Reply:
In previous default configs, we have 8M * 4 raft wal buffer for a partition, that would be 32M * 1024 = 32G (if we have import lots of data). So I change to 8M * 2 * 100 = 1.6G instead.